tailwind.config.js

```diff
@@ -1,5 +1,80 @@
+const screens = {
+  sm: "640px",
+  md: "768px",
+  lg: "1024px",
+  xl: "1280px",
+}
+
 module.exports = {
-  theme: {},
-  variants: {},
-  plugins: [],
+  theme: {
+    screens,
+    media: {
+      sm: `(min-width: ${screens.sm})`,
+      md: `(min-width: ${screens.md})`,
+      lg: `(min-width: ${screens.lg})`,
+      xl: `(min-width: ${screens.xl})`,
+    },
+    extend: {
+      spacing: {
+        "2d5": "0.625rem", // 10px
```

Comment:
@maxgerber I think here we can use dots, i.e. `"2.5": "0.625rem"`.

Reply:
Only thing is, is it weird to put a ```.``` within a class name considering that's how they're selected in CSS?

Reply:
So I think what happens with this is that the class is written to css as - 
```css
.mx-2\.5 {
  background-color: red;
}
```
But can be used in `html` as `className="mx-2.5"`

I know what you mean about it feeling strange though so happy to go with your decision.